src/demo/pages/BlockchainTx.tsx

```diff
@@ -113,221 +52,112 @@ export default function BlockchainTx() {
       return;
     }
 
-    onRender();
+    // onRender();
```

Comment:
remove comment. does this mean we can remove `onRender()`?

Reply:
ahh i will uncomment did while testing